src/assessments/audio-video-only/test-steps/audio-only-equivalent.tsx

```diff
@@ -5,7 +5,7 @@ import { link } from '../../../content/link';
 import * as content from '../../../content/test/audio-video-only/audio-only-equivalent';
 import { ManualTestRecordYourResults } from '../../common/manual-test-record-your-results';
 import * as Markup from '../../markup';
-import { TestStep } from '../../types/test-step';
+import { Requirement } from '../../types/requirement';
 import { AudioVideoOnlyTestStep } from './test-steps';
```

Comment:
Similarly, a future separate PR should rename the test-steps folders to requirements, and the contained test-steps.ts files to requirements.ts.